src/main/java/core/selenium/WebElementAction.java

```diff
@@ -216,4 +219,35 @@ public void selectByAction(final WebElement webElement) {
         Action action = builder.click(webElement).build();
         action.perform();
     }
+
+    /**
+     * Clicks on a WebElement.
+     *
+     * @param element for click
+     */
+    public void clickElement(final WebElement element) {
+        wait.until(ExpectedConditions.visibilityOf(element));
```

Comment:
Use the singleton instead of the wait

Reply:
ok It was solved  using waitForVisible method